examples/wallet/app/utils/wasmWrapper.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import { AccountState } from '../reducers/types';
+
+const wasmBindings = import('js-chain-libs/js_chain_libs');
+
+export default async function getAccountFromPrivateKey(
+  secret: string
+): AccountState {
+  const {
+    PrivateKey,
+    PublicKey,
+    Account,
+    AccountIdentifier,
+    AddressDiscrimination
+  } = await wasmBindings;
+  const privateKey: PrivateKey = PrivateKey.from_bech32(secret);
+  const publicKey: PublicKey = privateKey.to_public();
+  const account: Account = Account.from_public_key(publicKey);
+  const identifier: AccountIdentifier = account.to_identifier();
+  // FIXME: make the discrimination configurable
+  // FIXME somebody please tell me how do we determine the prefix
```

Comment:
You can't really determine it, I think, so I guess making it configurable is the only option (either statically or dynamically). But @NicolasDP can probably provide more insight

Reply:
leave the user  set anything they want, by default: `capu_rocks_` is a good prefix 🥇

Reply:
okay, it'll be configurable via a settings file. Since there are several hard-coded constants already, i think it should be the next issue for me to work on